### PR TITLE
Bump plutus version

### PR DESCRIPTION
**Pull request by szg251**
- Updated `cardano-wallet` to latest version (compatible with cardano-node 8.1.1)
- Updated nixpkgs, haskell.nix (moved to iog branch) , hackage.nix, iohk.nix

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -69,115 +63,75 @@
...
-              wrapProgram $out/bin/local-cluster \
-                --prefix PATH : "${lib.makeBinPath [
-                  config.hsPkgs.cardano-cli.components.exes.cardano-cli
-                  config.hsPkgs.cardano-node.components.exes.cardano-node
-                ]}"
-            '';
-          };
-        })
-      ];
-      extraSources = [
```
Comment (szg251) [inline flake.nix:89]: `extraSources` does not exist in the iohk haskell-nix, so everything here has to come either from CHap, cabal.project or a flake input

File: src/Plutip/Launch/Cluster.hs [hunk 2]
```diff
@@ -1,10 +1,12 @@
 {-# LANGUAGE DataKinds #-}
```
Comment (szg251) [inline src/Plutip/Launch/Cluster.hs:1]: Updated this file using the original in cardano-wallet, while preserving all our custom changes

**Review by zmrocze — APPROVED** [review]
looks all good

will make a release once i make it work in ctl with your changes of [#1532](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1532)

**Comment by zmrocze** [comment]
seems to work with: https://github.com/Plutonomicon/cardano-transaction-lib/pull/1573

merging and making release `plutip-w-node-8.1.1`

**Pull request merged**